tools/make/common.mk

```diff
@@ -30,6 +30,16 @@ ifeq ($(origin OUTPUT_DIR),undefined)
 OUTPUT_DIR := $(ROOT_DIR)/bin
 endif
 
+# Set Kubernetes Resources Directory Path
```

Comment:
* can we move these into kube.mk
* prefer if we named vars to KUBE_PROVIDER_DIR and KUBE_INFRA_DIR